server/graphql/types/station.js

```diff
@@ -27,11 +29,24 @@ export const resolvers = {
     longitude ({ stop_lon }) {
       return stop_lon
     },
-    routes ({ stop_id }, {from, length}) {
-      return db().query(aql`
+    routes ({ stop_id }, { from, length }) {
+      return RoutesByStationIdLoader.load({ stop_id, from, length })
+    }
+  }
+}
+
+const getRoutesByStationIds = (stopIds) => {
+  const { from, length } = stopIds[0]
+  stopIds = map(stopIds, 'stop_id')
+
+  return new Promise((resolve, reject) => {
```

Comment:
J'ai loggé cette requête côté arangodb

`
 for stopId in @value0
        let from = stopId.from

        let stops = (
          for stop in stops
          filter stop.parent_station == stopId
          return stop.stop_id
        )

        let stop_times_of_stop = (
          for stop_time in stop_times
          filter stop_time.stop_id in stops
          return stop_time.trip_id
        )
        let routes_of_stops = (
          for trip in trips
          filter trip.trip_id in stop_times_of_stop
          return trip.route_id
        )
        let routes = (
          for route in routes
          filter route.route_id in routes_of_stops
          limit @value1, @value2
          return route
        )
      return routes
`